File: librarymanager/librarymanager-be/src/main/java/ru/elenacherev/librarymanager/api/dto/Order.java
package ru.elenacherev.librarymanager.api.dto;

import lombok.Builder;
import lombok.Getter;
import ru.elenacherev.librarymanager.util.OrderState;

import java.util.Date;

@Builder
@Getter
public class Order {
    private Long orderId;
    private Date completeDate; //дата выполнения заказа (когда его выдали)
    private Date orderDate; //дата создания заказа
    private OrderState orderState; // состояние заказа
    private String readerFio;
    private String editionTitle;
    private Long readerId; //кто книгу заказал
    private Long editionId; //какую книгу заказал
}

File: librarymanager-api/src/main/java/ru/elenacherev/librarymanager/api/dto/Order.kt
package ru.elenacherev.librarymanager.api.dto

import ru.elenacherev.librarymanager.api.enum.OrderState
import java.util.*

data class Order(
    var orderId: UUID? = null,
    val completeDate: Date?,
    val orderDate: Date,
    val orderState: OrderState,
    val readerFio: String,
    val editionTitle: String,
    val readerId: UUID,
    val editionId: UUID
)